gatilegrid/tilegrids.py

```diff
@@ -321,6 +326,32 @@ def getClosestZoom(self, resolution, unit='meters'):
             return lo
         return lo - 1
 
+    def getCeilingZoom(self, resolution, unit='meters'):
+        """
+        Return the closest zoom level for a given resolution
+        Parameters:
+            resolution -- max. resolution
+            unit -- unit for output (default='meters')
+        """
+        assert unit in ('meters', 'degrees')
+        if unit == 'meters' and self.unit == 'degrees':
+            resolution = resolution / self.metersPerUnit
+        elif unit == 'degrees' and self.unit == 'meters':
+            resolution = resolution * EPSG4326_METERS_PER_UNIT
+        lo = 0
+        hi = len(self.RESOLUTIONS)
+        while lo < hi:
+            mid = (lo + hi) // 2
+            if resolution > self.RESOLUTIONS[mid]:
+                hi = mid
+            else:
+                lo = mid + 1
+        if lo == 0:
+            return lo
+        if hi == len(self.RESOLUTIONS):
+            return hi - 1
```

Comment:
Until here the code is exactly the same.
I think you can probably avoid to dulpicate code.

Reply:
I agree, thx for your feedback! I've put the binary search in a separate function now.